src/components/action-bar/action-bar.tsx

```diff
@@ -158,10 +204,16 @@ export class ActionBar implements ConditionalSlotComponent {
     connectConditionalSlotComponent(this);
   }
 
+  async componentWillLoad(): Promise<void> {
+    await setUpMessages(this);
```

Comment:
@jcfranco now that we're delaying `componentDidLoad` more with fetching async messages, I wonder if we need to prioritize updating our `setFocus()` methods to wait for the component to be rendered the first time before proceeding with focusing?

#5369 is a priority for MapViewer and I'd like to try and fix it and make them not have to increase any timeouts they have currently for focusing.